wrapanapi/systems/ec2.py

```diff
@@ -784,6 +784,9 @@ def create_s3_bucket(self, bucket_name):
             self.logger.exception("Error: Bucket was not successfully created.")
             return False
 
+    def list_s3_bucket(self):
```

Comment:
REQUIRED:  s3buckets ?
and I'd even call it list_s3_bucket_names, since it returns only list of names

Reply:
Update it.